sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_hourly_v1/query.sql

```diff
@@ -5,12 +5,12 @@ legacy_deduplicated_pings AS (
     *
   FROM
     `moz-fx-data-shared-prod.activity_stream_live.impression_stats_v1`
+    -- live is fine for backfills within the past 30 days, otherwise use stable
+    --`moz-fx-data-shared-prod.activity_stream_stable.impression_stats_v1`
```

Comment:
You might want to throw one of these at the bottom of this query

```
WHERE
      IF(
        DATE_DIFF(CURRENT_DATE(), @submission_date, DAY) > 30,
        ERROR("Data older than 30 days has been removed"),
        NULL
      ) IS NULL
```

Reply:
HOLY COW YOU CAN RAISE ERRORS IN BIGQUERY!?